Tests/UnitTests/Security/SigningTests.swift

```diff
@@ -310,7 +310,7 @@ class SigningTests: TestCase {
         curl -v 'https://api.revenuecat.com/v1/subscribers/login' \
         -X GET \
         -H 'X-Nonce: MTIzNDU2Nzg5MGFi' \
-        -H 'Authorization: Bearer {api_key}'
+        -H 'Authorization: Bearer appl_fFVBVAoYujMZJnepIziGKVjnZBz'
```

Comment:
This is already exposed in the constant below, so makes it easier to copy these `curl` commands and ensure that you use the right key.